posthog/api/decide.py

```diff
@@ -92,10 +92,23 @@ def get_decide(request: HttpRequest):
         team = Team.objects.get_team_from_token(token)
         if team is None and token:
             project_id = _get_project_id(data, request)
+
+            if not project_id:
+                return cors_response(
+                    request,
+                    JsonResponse(
+                        {
+                            "code": "validation",
+                            "message": "Project API key invalid. You can find your project API key in PostHog project settings.",
+                        },
+                        status=400,
+                    ),
+                )
+
             user = User.objects.get_from_personal_api_key(token)
             if user is None:
                 return cors_response(
-                    request, JsonResponse({"code": "validation", "message": "Invalid personal API key.",}, status=400,),
+                    request, JsonResponse({"code": "validation", "message": "Personal API key invalid.",}, status=400,),
```

Comment:
very much a nit: maybe a 401 is more appropriate given that this is an authentication error?

Reply:
I was going for consistency with /capture endpoint but you're right!